documentation/book/ref-list-of-kafka-mirror-maker-resources.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+[id='ref-list-of-kafka-mirror-maker-resources-{context}']
+= List of resources created as part of Kafka Mirror Maker
+
+The following resources will created by the Cluster Operator in the {ProductPlatformName} cluster:
+
+_<mirror-maker-name>_-mirror-maker:: Deployment which is in charge to create the Kafka Mirror Maker pods.
```

Comment:
`Deployment` not Deployment

Reply:
We are not using the monospace for any of other ref list, I wouldn't end up with inconsistent doc. Do I have to change all the other or opening an issue remembering about that. It's the same for many other comments which I have addressed but can be applied to other parts (i.e. Kafka Connect doc).